(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[8204],{5669:function(i,e,t){(window.__NEXT_P=window.__NEXT_P||[]).push(["/tutorials",function(){return t(4591)}])},1514:function(i,e){"use strict";e.Z=[{title:"About Neural ODE : Using Torchdiffeq with Deepchem\xb6",urlifiedFileName:"about-node-using-torchdiffeq-in-deepchem"},{title:"Advanced Model Training\xb6",urlifiedFileName:"advanced-model-training"},{title:"Advanced model training using hyperopt\xb6",urlifiedFileName:"advanced-model-training-using-hyperopt"},{title:"An Introduction To MoleculeNet\xb6",urlifiedFileName:"an-introduction-to-moleculenet"},{title:"Calculating Atomic Contributions for Molecules Based on a Graph Convolutional  QSAR Model\xb6",urlifiedFileName:"atomic-contributions-for-molecules"},{title:"Conditional Generative Adversarial Network\xb6",urlifiedFileName:"conditional-generative-adversarial-networks"},{title:"Creating Models with TensorFlow and PyTorch\xb6",urlifiedFileName:"creating-models-with-tensorflow-and-pytorch"},{title:"Creating a High Fidelity Dataset from Experimental Data\xb6",urlifiedFileName:"creating-a-high-fidelity-model-from-experimental-data"},{title:"Colab\xb6",urlifiedFileName:"deepqmc-tutorial"},{title:"Deep Probabilistic Analysis of Single-Cell Omics Data\xb6",urlifiedFileName:"deep-probabilistic-analysis-of-single-cell-omics-data"},{title:"Exploring Quantum Chemistry with GDB1k\xb6",urlifiedFileName:"exploring-quantum-chemistry-with-gdb1k"},{title:"Generating Molecules with MolGAN\xb6",urlifiedFileName:"generating-molecules-with-molgan"},{title:"Going Deeper On Molecular Featurizations\xb6",urlifiedFileName:"going-deeper-on-molecular-featurizations"},{title:"Hierarchial Generation of Moelcules\xb6",urlifiedFileName:"hierarchial-moelcular-generation-with-hiervae"},{title:"Interactive model evaluation with Trident Chemwidgets\xb6",urlifiedFileName:"interactive-model-evaluation-with-trident-chemwidgets"},{title:"Given Physical Data\xb6",urlifiedFileName:"introducing-jaxmodel-and-pinnmodel"},{title:"Introduction To Material Science\xb6",urlifiedFileName:"introduction-to-material-science"},{title:"Introduction to Bioinformatics\xb6",urlifiedFileName:"introduction-to-bioinformatics"},{title:"Introduction to GROVER\xb6",urlifiedFileName:"introduction-to-grover"},{title:"Introduction to Gaussian Processes\xb6",urlifiedFileName:"introduction-to-gaussian-processes"},{title:"Introduction to Graph Convolutions\xb6",urlifiedFileName:"introduction-to-graph-convolutions"},{title:"Introduction to Model Interpretability\xb6",urlifiedFileName:"introduction-to-model-interpretability"},{title:"Introduction to the Molecular Attention Transformer.\xb6",urlifiedFileName:"introduction-to-molecular-attention-transformer"},{title:"Screening Zinc For HIV Inhibition\xb6",urlifiedFileName:"large-scale-chemical-screens"},{title:"Learning Unsupervised Embeddings for Molecules\xb6",urlifiedFileName:"learning-unsupervised-embeddings-for-molecules"},{title:"Tutorial Part 13: Modeling Protein-Ligand Interactions\xb6",urlifiedFileName:"modeling-protein-ligand-interactions"},{title:"Modeling Protein-Ligand Interactions with Atomic Convolutions\xb6",urlifiedFileName:"modeling-protein-ligand-interactions-with-atomic-convolutions"},{title:"Molecular Fingerprints\xb6",urlifiedFileName:"molecular-fingerprints"},{title:"Congratulations! Time to join the Community!\xb6",urlifiedFileName:"multisequence-alignments"},{title:"Tutorial 33: Physics Informed Neural Networks using JaxModel & PINN_Model\xb6",urlifiedFileName:"physics-informed-neural-networks"},{title:"Protein Deep Learning\xb6",urlifiedFileName:"protein-deep-learning"},{title:"Putting Multitask Learning to Work\xb6",urlifiedFileName:"putting-multitask-learning-to-work"},{title:"Pytorch-Lightning Integration for DeepChem Models\xb6",urlifiedFileName:"pytorchlightning-integration"},{title:"ScanPy\xb6",urlifiedFileName:"scanpy"},{title:"Synthetic Feasibility\xb6",urlifiedFileName:"synthetic-feasibility-scoring"},{title:"The Basic Tools of the Deep Life Sciences\xb6",urlifiedFileName:"the-basic-tools-of-the-deep-life-sciences"},{title:"Training a Generative Adversarial Network on MNIST\xb6",urlifiedFileName:"training-a-generative-adversarial-network-on-mnist"},{title:"Tutorial Molecular Machine Learinig Tutorial 7 : Training a Normalizing Flow on QM9\xb6",urlifiedFileName:"training-a-normalizing-flow-on-qm9"},{title:"Tutorial: ChemBERTa: Large-Scale Self-Supervised Pretraining for Molecular Property Prediction using a Smiles Tokenization Strategy\xb6",urlifiedFileName:"transfer-learning-with-chemberta-transformers"},{title:"Uncertainty in Deep Learning\xb6",urlifiedFileName:"uncertainty-in-deep-learning"},{title:"Using Reinforcement Learning to Play Pong\xb6",urlifiedFileName:"using-reinforcement-learning-to-play-pong"},{title:"Working With Datasets\xb6",urlifiedFileName:"working-with-datasets"},{title:"Working With Splitters\xb6",urlifiedFileName:"working-with-splitters"}]},4591:function(i,e,t){"use strict";t.r(e),t.d(e,{default:function(){return a}});var n=t(5893),l=t(1514),r=t(1163),o=t(7294);function a(){let i=(0,r.useRouter)(),e=l.Z[0];return(0,o.useEffect)(()=>{i.push("tutorials/".concat(e.urlifiedFileName.replace(/\.[^/.]+$/,"")))}),(0,n.jsx)(n.Fragment,{})}},1163:function(i,e,t){i.exports=t(880)}},function(i){i.O(0,[9774,2888,179],function(){return i(i.s=5669)}),_N_E=i.O()}]);