(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[8204],{5669:function(e,i,t){(window.__NEXT_P=window.__NEXT_P||[]).push(["/tutorials",function(){return t(4591)}])},1514:function(e,i){"use strict";i.Z=[{title:"About Neural ODE : Using Torchdiffeq with Deepchem\xb6",urlifiedFileName:"About-nODE-Using-Torchdiffeq-in-Deepchem"},{title:"Advanced Model Training\xb6",urlifiedFileName:"Advanced-Model-Training"},{title:"Advanced model training using hyperopt\xb6",urlifiedFileName:"Advanced-model-training-using-hyperopt"},{title:"An Introduction To MoleculeNet\xb6",urlifiedFileName:"An-Introduction-To-MoleculeNet"},{title:"Calculating Atomic Contributions for Molecules Based on a Graph Convolutional  QSAR Model\xb6",urlifiedFileName:"Atomic-Contributions-for-Molecules"},{title:"Conditional Generative Adversarial Network\xb6",urlifiedFileName:"Conditional-Generative-Adversarial-Networks"},{title:"Creating Models with TensorFlow and PyTorch\xb6",urlifiedFileName:"Creating-Models-with-TensorFlow-and-PyTorch"},{title:"Creating a High Fidelity Dataset from Experimental Data\xb6",urlifiedFileName:"Creating-a-high-fidelity-model-from-experimental-data"},{title:"Deep Probabilistic Analysis of Single-Cell Omics Data\xb6",urlifiedFileName:"Deep-probabilistic-analysis-of-single-cell-omics-data"},{title:"Exploring Quantum Chemistry with GDB1k\xb6",urlifiedFileName:"Exploring-Quantum-Chemistry-with-GDB1k"},{title:"Generating Molecules with MolGAN\xb6",urlifiedFileName:"Generating-molecules-with-MolGAN"},{title:"Going Deeper On Molecular Featurizations\xb6",urlifiedFileName:"Going-Deeper-on-Molecular-Featurizations"},{title:"Hierarchial Generation of Moelcules\xb6",urlifiedFileName:"Hierarchial-Moelcular-Generation-with-HierVAE"},{title:"Interactive model evaluation with Trident Chemwidgets\xb6",urlifiedFileName:"Interactive-Model-Evaluation-with-Trident-Chemwidgets"},{title:"Given Physical Data\xb6",urlifiedFileName:"Introducing-JaxModel-and-PINNModel"},{title:"Introduction To Material Science\xb6",urlifiedFileName:"Introduction-To-Material-Science"},{title:"Introduction to Bioinformatics\xb6",urlifiedFileName:"Introduction-to-Bioinformatics"},{title:"Introduction to GROVER\xb6",urlifiedFileName:"Introduction-to-GROVER"},{title:"Introduction to Gaussian Processes\xb6",urlifiedFileName:"Introduction-to-Gaussian-Processes"},{title:"Introduction to Graph Convolutions\xb6",urlifiedFileName:"Introduction-to-Graph-Convolutions"},{title:"Introduction to Model Interpretability\xb6",urlifiedFileName:"Introduction-to-Model-Interpretability"},{title:"Introduction to the Molecular Attention Transformer.\xb6",urlifiedFileName:"Introduction-to-Molecular-Attention-Transformer"},{title:"Screening Zinc For HIV Inhibition\xb6",urlifiedFileName:"Large-Scale-Chemical-Screens"},{title:"Learning Unsupervised Embeddings for Molecules\xb6",urlifiedFileName:"Learning-Unsupervised-Embeddings-for-Molecules"},{title:"Modeling Protein-Ligand Interactions with Atomic Convolutions\xb6",urlifiedFileName:"Modeling-Protein-Ligand-Interactions-With-Atomic-Convolutions"},{title:"Molecular Fingerprints\xb6",urlifiedFileName:"Molecular-Fingerprints"},{title:"Congratulations! Time to join the Community!\xb6",urlifiedFileName:"Multisequence-Alignments"},{title:"Tutorial 33: Physics Informed Neural Networks using JaxModel & PINN_Model\xb6",urlifiedFileName:"Physics-Informed-Neural-Networks"},{title:"Protein Deep Learning\xb6",urlifiedFileName:"Protein-Deep-Learning"},{title:"Putting Multitask Learning to Work\xb6",urlifiedFileName:"Putting-Multitask-Learning-to-Work"},{title:"Pytorch-Lightning Integration for DeepChem Models\xb6",urlifiedFileName:"PytorchLightning-Integration"},{title:"ScanPy\xb6",urlifiedFileName:"Scanpy"},{title:"Synthetic Feasibility\xb6",urlifiedFileName:"Synthetic-Feasibility-Scoring"},{title:"The Basic Tools of the Deep Life Sciences\xb6",urlifiedFileName:"The-Basic-Tools-of-the-Deep-Life-Sciences"},{title:"Training a Generative Adversarial Network on MNIST\xb6",urlifiedFileName:"Training-a-Generative-Adversarial-Network-on-MNIST"},{title:"Tutorial Molecular Machine Learinig Tutorial 7 : Training a Normalizing Flow on QM9\xb6",urlifiedFileName:"Training-a-Normalizing-Flow-on-QM9"},{title:"Uncertainty in Deep Learning\xb6",urlifiedFileName:"Uncertainty-In-Deep-Learning"},{title:"Using Reinforcement Learning to Play Pong\xb6",urlifiedFileName:"Using-Reinforcement-Learning-to-Play-Pong"},{title:"Working With Datasets\xb6",urlifiedFileName:"Working-With-Datasets"},{title:"Working With Splitters\xb6",urlifiedFileName:"Working-With-Splitters"}]},4591:function(e,i,t){"use strict";t.r(i),t.d(i,{default:function(){return a}});var n=t(5893),l=t(1514),o=t(1163),r=t(7294);function a(){let e=(0,o.useRouter)(),i=l.Z[0];return(0,r.useEffect)(()=>{e.push("tutorials/".concat(i.urlifiedFileName.replace(/\.[^/.]+$/,"")))}),(0,n.jsx)(n.Fragment,{})}},1163:function(e,i,t){e.exports=t(880)}},function(e){e.O(0,[9774,2888,179],function(){return e(e.s=5669)}),_N_E=e.O()}]);